src/General-Rules-Tests/ReInconsistentMethodClassificationRuleTest.class.st

```diff
@@ -0,0 +1,25 @@
+Class {
+	#name : 'ReInconsistentMethodClassificationRuleTest',
+	#superclass : 'ReAbstractRuleTestCase',
+	#category : 'General-Rules-Tests-Migrated',
+	#package : 'General-Rules-Tests',
+	#tag : 'Migrated'
+}
+
+{ #category : 'accessing' }
+ReInconsistentMethodClassificationRuleTest >> initialize [ 
+]
```

Comment:
We have a release test that ensure all #initialize methods are in the right protocol and this makes the test fail. 

Instead of having this method here, we could generate the method like this:

```st
self class compile: 'initialize super initialize' classified: 'accessing'.
[ "code of the test" ] ensure: [ (self class >> #initialize) removeFromSystem ]
```

#ensure: will make sure that even if the test crashes, the block deleting the method will be executed.